layouts/default.vue

```diff
@@ -99,7 +99,11 @@ body {
 
 html,
 body {
-  background-color: #181829;
+  background-color: $background-dark;
+}
+
+.app {
```

Comment:
Maybe instead of `.app` we could call it `.mobile-background`, I feel like it's clearer?

Reply:
Resolved as this is for the app view only